### DISCUSSION TITLE
Choosing what is displayed on the "Cable Trace"

### DISCUSSION THREAD

**Discussion by KaricAllegra** [netbox-community/netbox#18240] [Help Wanted!] [answered]
Currently running NetBox Community v4.1.7

Is there a way to change what is displayed on the trace? For example, if I want to show the Tags I've attached to my cables on the trace, how do I enable them to be displayed?

Thanks for any help, and sorry if I missed this somewhere obvious lol.

**Comment by candlerb** [comment, marked as answer]
It's not customizable; you'd have to raise a feature request. For example #12831

**Reply by KaricAllegra** [reply]
Copy that. Glad it wasn't just something I had missed. Thank you!
